app/views/videos/index.html.haml

```diff
@@ -3,7 +3,7 @@
     %header
       = link_to category_path(category) do
         %h3= category.name.upcase
-      .videos.row
+      .video.row
```

Comment:
I'm not seeing anything either. It may have another purpose that I don't recall off the top of my head. The more important idea here is that you are actively programming but passively implementing the style/html structure you are given.